docs/UserGuide.md

```diff
@@ -5,7 +5,18 @@ pageNav: 3
 ---
 
 # Internhub User Guide
-Intern Hub is a desktop app for managing contacts, optimized for use via a Command Line Interface (CLI) while still offering the benefits of a Graphical User Interface (GUI). If you can type fast, InternHub can help you complete your contact management tasks more efficiently than traditional GUI apps. Intern Hub is designed for undergraduates, professors, and carreer guidance officers seeking a streamlined approach to managing internship contacts.Intern Hub simplifies the process of organizing and accessing contact information, making it ideal for users who value efficiency and productivity.
+
+## Welcome !
+Are you an undergraduate navigating the maze of internship opportunities? 
+Look no further! InternHub is here to revolutionize the way you manage your internship contacts.
+This User Guide (UG) serves as a comprehensive resource to aid users in understanding and effectively utilizing the application.
+It provides detailed instructions on installation, usage, and navigation, ensuring users can maximize the benefits of Intern Hub for managing internship contacts.
+
+## Who can use InternHub ?
+InternHub is designed for undergraduates, professors, and career guidance officers seeking a streamlined approach to managing internship contacts.
+InternHub is crafted with the intention of being user-friendly for individuals with varying levels of computer experience.
+We trust that you'll find the interface intuitive and easy to navigate, whether you're a tech-savvy enthusiast or just starting to explore desktop applications. 
+We're here to support you every step of the way on your internship management journey!
```

Comment:
Great introduction! It effectively highlights the purpose and target audience of InternHub, setting clear expectations for users. The tone is welcoming and inclusive, which is excellent for engaging users from different backgrounds.